README.md

```diff
@@ -18,41 +18,66 @@ It's used in the [Snap framework](http://github.com/intelsdi-x/snap).
-You can get the pre-built binaries for your OS and architecture under the plugin's [release](https://github.com/intelsdi-x/snap-plugin-collector-pysmart/releases) page.  For Snap, check [here](https://github.com/intelsdi-x/snap/releases).
 
+#### Python module
 
-#### To build the plugin binary:
-Fork https://github.com/intelsdi-x/snap-plugin-collector-pysmart
+The preferred way to run Python based plugins is to leverage the python package
+index.  This means that the Snap daemon, *snapteld*, will need to be run in an
+environment where the python module `snap-plugin-collector-pysmart` was installed.
+If you install the plugin (`pip install snap-plugin-collector-pysmart`) into the
+system's Python environment that should be enough.  If you use a
+[virtualenv](https://pypi.python.org/pypi/virtualenv) be sure to activate it 
+before starting `snapteld` since it will need access to the plugin and its
+dependencies.
 
-Clone repo into `$GOPATH/src/github.com/intelsdi-x/`:
+To install the plugin run: 
+* `pip install snap-plugin-collector-pysmart`
+
+The plugin includes a command line entry point also called
+`snap-plugin-collector-pysmart` which should be in your path after installation.
+
+Find the plugin's command line script:
+* `which snap-plugin-collector-pysmart`
 
 ```
-$ git clone https://github.com/<yourGithubID>/snap-plugin-collector-pysmart.git
+which snap-plugin-collector-pysmart
+> /Users/jrcookli/.pyenv/versions/py2712/bin/snap-plugin-collector-pysmart
 ```
 
-### Configuration and Usage
-* Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
-* [Snap Python documentation](https://intelsdi-x.github.io/snap-plugin-lib-py/index.html)
+The entry point script, snap-plugin-collector-pysmart, is what we will be
+load into `snapteld` with the following command.
+
+```
+snaptel plugin load `which snap-plugin-collector-pysmart`
+```
 
-## Documentation
+#### Plugin package
 
-There are a number of other resources you can review to learn to use this plugin:
-* [Snap pysmart examples](#examples)
+An alternative installation method for Linux X86_64 is to use the binary package.
+The package includes a Python 2.7 distribution with the plugin already installed.
 
-### Collected Metrics
-This plugin will identify all the devices on the node which have [S.M.A.R.T.](https://www.smartmontools.org/) enabled and automatically populate the list of collected metrics based on which are being exposed by the device. This will be different per manufacturer and per device. 
+You can get the pre-built plugin package under
+[releases](https://github.com/intelsdi-x/snap-plugin-collector-pysmart/releases)
+page.
 
-* Ensure that S.M.A.R.T. is enabled on the device
+Known issues:
+* Longer plugin load times 
+* A package is only available for Linux X86_64
+
+Since the current default timeout may be exceeded start `snapteld` with the flag
+`--plugin-load-timeout 30` (e.g. `snapteld -t 0 -1 --plugin-load-timeout 30`).
```

Comment:
Minor typo:  Missing "l" option (log level) in `snapteld -t 0 -1 --plugin-load-timeout 30`.